Add Contributor Growth working group

carolynvs opened the pull request:
Makes the Contributor Growth working group official!

I wasn't sure who all is in the subproject, so if you don't see your name but think you are in the subproject. YOU ARE IN! 🙌 I just went by who had something assigned to them on #12. I can add you now in this PR, or you can PR yourself in later.

I am assuming that merging this PR will mean that the subproject was accepted. That's why I made the edit to the README at the same time. If that's not right, please let me know and I'll switch to marking it as proposed instead like the others.

Closes #12

carolynvs commented:
I don't know why I can't add @karenhchu as a reviewer. She is in the settings.yaml, is there more that we need to do?

karenhchu commented on contributor-growth/README.md:
```diff
@@ -0,0 +1,65 @@
+
+* Provide best practices and resources for managing and interacting with
+  contributors to CNCF projects.
+* Provide guidance on building a contributor pipeline that brings in new
```
Do we want to consider providing guidance on how to handle transitioning maintainers to emeritus status? It's not so much for growth but it can affect the sustainability of contributing.

karenhchu commented on contributor-growth/README.md:
```diff
@@ -0,0 +1,65 @@
+* Project governance.
+* Project automation, such as prow.
+* Day to day support for improving the contribution experience for CNCF
+  projects. That falls under other projects, for example Kubernetes SIG
```
Can we clarify what "other projects" mean? Do we mean within a CNCF project's own scope, with the example being K8s SIG Contributor Experience?

carolynvs commented:
@karenhchu I've updated it to address your comments. 👍

karenhchu commented on contributor-growth/README.md:
```diff
@@ -0,0 +1,71 @@
+* Project automation, such as prow.
+* Day to day support for improving the contribution experience for CNCF
+  projects. Some CNCF projects have support for this, for example within
+  Kubernetes, that would be Kubernetes SIG Contributor Experience. Even when
```
I'm still a bit confused by this line -- "Even when there is not a relevant CNCF project these activities are out-of-scope for this subproject." Can we clarify this to make sure there's only one interpretation of it? 

Correct me if I'm wrong but I believe we're trying to say that some projects take on the day to day support themselves (as, an example, through the K8s SIG) but in the absence of a project providing its own day to day support, that does not mean the burden falls onto this group. 

If so, how about -- "Day to day support for improving the contribution experience for CNCF projects. Instead, CNCF projects can take on this support themselves (for example, the CNCF project Kubernetes has its own SIG Contributor Experience)."

I'm hoping by stating what _they_ can take on, we eliminate ourselves of that responsibility. Does that make sense? 😅

carolynvs replied:
Yes, thank you that _is_ what I am trying to say. Let me update it with your wording. 👍

carolynvs commented:
* I have updated it to use Karen's wording (with a small change) for making sure activities like K8's SIG ContribEX is out of scope.
* I have updated it to use the term working group instead of subproject (see #27).

parispittman commented:
i have some things to add but all can go in a subsequent PR thank you @carolynvs for the work here. ie adding the word operations to the out of scope line will clarify the support we aren't willing to provide (day to day operational support)

the only thing i'd like to change now is "create spaces for maintainers of CNCF projects to share their own best practices and ask questions" as thats a separate working group for maintainers circle

jberkus commented:
/lgtm

carolynvs commented:
* Rebased to fix the conflict with master
* Removed the mention of maintainer spaces now that maintainer's circle is a separate WG

carolynvs commented:
@parispittman @justaugustus I'm not sure what else needs to happen before we can merge this. 🤔 Let me know who else I should tag in for review!

jberkus commented:
@carolynvs last I heard, you were waiting for @karenhchu approval.

karenhchu commented:
/lgtm :)
